packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1062,6 +1064,17 @@ class EditableText extends StatefulWidget {
   /// {@macro flutter.rendering.editable.selectionEnabled}
   bool get selectionEnabled => enableInteractiveSelection;
 
+  /// {@template flutter.widgets.editableText.autofillHints}
+  /// A list of strings that helps the autofill service identify the type of this
+  /// text input.
+  ///
+  /// When set to null, the text input will not participate in autofill triggered
+  /// by a different [AutofillClient], even if they're in the same [AutofillScope].
+  ///
+  /// {@macro flutter.services.autofill.autofillHints}
+  /// {@endtemplate}
+  final Iterable<String> autofillHints;
```

Comment:
Why is this an Iterable and not specifically a List?

Reply:
The hints collection does not need random access.